e2e/to-match-inline-snapshot/__tests__/basic-support.test.js

```diff
@@ -0,0 +1,6 @@
+test('inline snapshots', () =>
```

Comment:
this file should not be checked in

Reply:
Woops, I may have miss clicked when adding the last commit

Reply:
Will update

Reply:
Fixed